### ISSUE TITLE
Add timeframe drop down to filter CVEs, USNs by date

### ISSUE THREAD

**Issue by Khutch25** [canonical/ubuntu.com#9497]
Request from sales and a valid point, can a timeframe filter be included on the /security/notices https://ubuntu.com/security/cve?q=&package=&priority=critical&version=&status=

I.e show me the critical CVE's in the last 18 months 

---
*Reported from: https://ubuntu.com/security/cve?q=

**Comment by petesfrench** [comment]
@mtruj013 Do you know if this was considered in the CVE overhaul?

**Comment by mtruj013** [comment]
Yes, but it was de-scoped for MVP so we're not including it for now. Not sure if we have a corresponding jira ticket for the request, but I'll make one if not

**Comment by muhammad-ali-pk** [comment]
@mtruj013 Any idea if there's been any progress on this as of yet?
